addons/binding/org.openhab.binding.openuv/src/main/java/org/openhab/binding/openuv/handler/OpenUVBridgeHandler.java

```diff
@@ -0,0 +1,164 @@
+/**
+ * Copyright (c) 2010-2018 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.openuv.handler;
+
+import static org.openhab.binding.openuv.OpenUVBindingConstants.BASE_URL;
+
+import java.io.IOException;
+import java.time.ZonedDateTime;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Properties;
+
+import org.apache.commons.lang.StringUtils;
+import org.eclipse.jdt.annotation.NonNull;
+import org.eclipse.jdt.annotation.Nullable;
+import org.eclipse.smarthome.config.core.Configuration;
+import org.eclipse.smarthome.core.library.types.DecimalType;
+import org.eclipse.smarthome.core.thing.Bridge;
+import org.eclipse.smarthome.core.thing.ChannelUID;
+import org.eclipse.smarthome.core.thing.ThingStatus;
+import org.eclipse.smarthome.core.thing.ThingStatusDetail;
+import org.eclipse.smarthome.core.thing.ThingUID;
+import org.eclipse.smarthome.core.thing.binding.BaseBridgeHandler;
+import org.eclipse.smarthome.core.types.Command;
+import org.eclipse.smarthome.io.net.http.HttpUtil;
+import org.openhab.binding.openuv.OpenUVBindingConstants;
+import org.openhab.binding.openuv.json.OpenUVJsonResponse;
+import org.osgi.framework.ServiceRegistration;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.JsonDeserializer;
+
+/**
+ * {@link OpenUVBridgeHandler} is the handler for OpenUV API and connects it
+ * to the webservice.
+ *
+ * @author Gaël L'hopital - Initial contribution
+ *
+ */
+public class OpenUVBridgeHandler extends BaseBridgeHandler {
```

Comment:
Null annotations are used but the class seems to be missing `@NonNullByDefault`.